clustered/src/main/java/consumer/RebootConsumer.java

```diff
@@ -0,0 +1,86 @@
+package consumer;
+
+import static commons.Config.BY;
+import static commons.Config.REBOOT_ADDRESS;
+import static commons.Config.STATUS;
+
+import java.util.UUID;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import io.vertx.core.Future;
+import io.vertx.core.VertxOptions;
+import io.vertx.core.json.JsonObject;
+import io.vertx.reactivex.core.AbstractVerticle;
+import io.vertx.reactivex.core.Vertx;
+import io.vertx.reactivex.core.shareddata.Lock;
+
+public class RebootConsumer extends AbstractVerticle {
+
+   private final Logger logger = Logger.getLogger(RebootConsumer.class.getName());
+   private boolean reboot = false;
+   private String id = "ID-" + UUID.randomUUID().toString().substring(0, 4).toUpperCase();
+
+   @Override
+   public void start(Future<Void> startFuture) throws Exception {
+      logger.info("Reboot verticle " + id + " started");
+      vertx.eventBus().consumer("ids", message -> {
+         int id = ((Integer) message.body()).intValue();
```

Comment:
I don't believe you need to cast if you use a typed consumer method. Like:

```java
vertx.eventBus().<Integer>consumer("ids", message -> {
  int id = message.body()
});
```

Reply:
yes, perfect, did not realise haha